doc/getstarted/index_en.rst

```diff
@@ -1,9 +1,66 @@
 GET STARTED
 ============
 
+.. _quick_install:
+
+Quick Install
+----------------------
+
+You can use pip to install PaddlePaddle using a single command, supports
```

Comment:
using->with

Reply:
Thanks for reviewing, comments all done.